README.md

```diff
@@ -727,10 +727,10 @@ Error Connecting to auth service ...
 }
 ```
 
-## Parse import specifications
+## Parse bulk specifications
```

Comment:
This endpoint isn't necessarily import specific - it could be used for bulk analysis as well. Updated the docs to be agnostic.